package/src/components/AttachmentPicker/AttachmentPicker.tsx

```diff
@@ -144,16 +138,22 @@ const renderImage = ({
     setSelectedImages,
   } = item;
 
-  const localUri = asset.id
-    ? getLocalAssetUri(asset.id)
-    : asset.uri?.match(/assets-library/)
-    ? getLocalAssetUri(asset.uri)
-    : asset.uri;
-  const uri = asset.filename || localUri || '';
-  const filename = uri.replace(/^(file:\/\/|content:\/\/|assets-library:\/\/)/, '');
-  const contentType = lookup(filename) || 'multipart/form-data';
+  const contentType = lookup(asset.filename) || 'multipart/form-data';
 
-  const isImage = contentType.startsWith('image/');
+  const isImage = asset.filename
+    ? contentType.startsWith('image/')
+      ? 'image'
+      : 'video'
+    : asset.type === 'video'
+    ? 'video'
+    : 'image';
+  const videoDuration = asset.duration ? asset.duration : asset.playableDuration;
+
+  const duration = videoDuration
+    ? videoDuration / 3600 >= 1
+      ? dayjs.duration(videoDuration, 'second').format('HH:mm:ss')
+      : dayjs.duration(videoDuration, 'second').format('mm:ss')
+    : null;
```

Comment:
same comment as in the other PR

Reply:
Solved in that PR. If that is merged it will be solved here to as this is created over that branch.